apps/functions-subscription/src/config.ts

```diff
@@ -0,0 +1,52 @@
+import * as t from 'io-ts';
+import * as PR from 'io-ts/PathReporter';
+import { pipe } from 'fp-ts/lib/function';
+import * as E from 'fp-ts/lib/Either';
+import { NonEmptyString } from '@pagopa/ts-commons/lib/strings';
+
+export interface Config {
+  readonly subscriptionRequest: {
+    readonly eventhub: {
+      readonly connectionString: string;
+      readonly name: string;
+    };
+  };
+  readonly subscription: {
+    readonly cosmosdb: {
+      readonly connectionString: string;
+      readonly databaseName: string;
+    };
+  };
```

Comment:
**question**: why not keeping a flatten structure?

I mean, what are the advantages of having this structure instead of something like
```typescript
readonly eventhub: {
  readonly connectionString: string;
  readonly name: string;
};
readonly cosmosdb: {
  readonly connectionString: string;
  readonly databaseName: string;
};
```

Reply:
We will have another `eventhub`, which is why it is not flattened. As for `cosmosdb`, we can flatten it because we will only have one db.